src/actions/LocalSettingsActions.ts

```diff
@@ -160,7 +160,7 @@ export const readLocalSettings = async (account: EdgeAccount): Promise<LocalAcco
   }
 }
 
-export const writeLocalSettings = async (account: EdgeAccount, settings: object) => {
+export const writeLocalSettings = async (account: EdgeAccount, settings: LocalAccountSettings) => {
   const text = JSON.stringify(settings)
```

Comment:
This is were you'd do the uncleaner to get the text: `wasLocalAccountSettings(settings)`. Again, optional.